src/main/kotlin/no/nav/syfo/service/SykmeldingService.kt

```diff
@@ -28,18 +27,23 @@ class SykmeldingService constructor(private val sykmeldingerConsumer: Sykmelding
         }
     }
 
-    suspend fun checkSykmeldingStatus(
-        varselDato: LocalDate,
-        fnr: String,
-        virksomhetsnummer: String?
+    suspend fun checkSykmeldingStatusForVirksomhet(
+        varselDato: LocalDate, fnr: String, virksomhetsnummer: String?
     ): SykmeldingStatus {
-        val sykmeldingerPaaVarseldato: List<SykmeldingDTO> = sykmeldingerConsumer.getSykmeldingerPaDato(varselDato, fnr)
-            ?: return SykmeldingStatus(isSykmeldtIJobb = false, sendtArbeidsgiver = false)
+        val sykmeldingerPaVarseldato: List<SykmeldingDTO> =
+            sykmeldingerConsumer.getSykmeldingerPaDato(varselDato, fnr) ?: return SykmeldingStatus(isSykmeldtIJobb = false, sendtArbeidsgiver = false)
 
-        val isSendtAG = isSendtAG(sykmeldingerPaaVarseldato, virksomhetsnummer)
+        val isSendtAG = isSendtAG(sykmeldingerPaVarseldato, virksomhetsnummer)
 
-        val isSykmeldtIJobb = isSykmeldtIJobb(sykmeldingerPaaVarseldato)
+        val isSykmeldtIJobb = isSykmeldtIJobb(sykmeldingerPaVarseldato)
 
         return SykmeldingStatus(isSykmeldtIJobb = isSykmeldtIJobb, sendtArbeidsgiver = isSendtAG)
     }
+
+    suspend fun isAktiveSykmeldingerPaVarseldato(varselDato: LocalDate, fnr: String): Boolean {
+        val sykmeldingerPaVarseldato = sykmeldingerConsumer.getSykmeldingerPaDato(varselDato, fnr)
+        val sendteSykmeldinger: List<SykmeldingDTO> = sykmeldingerPaVarseldato?.filter { sykmelding -> sykmelding.sykmeldingStatus.statusEvent == "SENDT" } ?: listOf()
+
+        return sendteSykmeldinger.isNotEmpty()
```

Comment:
Hm, blir det her riktig? Dersom status er "SENDT" betyr det at den er sendt arbeidsgiver.

Men den kan jo være aktiv selv om den ikke er sendt.

Slik er sjekken til Team Sykmelding for å vite om en person har aktiv sykmelding:
```
suspend fun getSykmeldtStatusForDato(fnr: String, dato: LocalDate): SykmeldtStatus {
        val sykmeldinger = getInternalSykmeldinger(fnr, dato, dato)
        return if (sykmeldinger.isEmpty()) {
            SykmeldtStatus(erSykmeldt = false, gradert = null, fom = null, tom = null)
        } else {
            val sykmelding = sykmeldinger.first()
            SykmeldtStatus(erSykmeldt = true, gradert = inneholderGradertPeriode(sykmelding.sykmeldingsperioder), fom = finnForsteFom(sykmelding.sykmeldingsperioder), tom = finnSisteTom(sykmelding.sykmeldingsperioder))
        }
    }

```

Så for vår del er det vel bare å sjekke om sykmeldingerPaVarseldato ikke er tom, tror jeg?

Reply:
Laget en POST-spøring til `/api/v2/sykmelding/sykmeldtStatus`